Product Management Service/Code/product-service/src/main/resources/db/migration/V1__init.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE products(
+    id int PRIMARY KEY ,
+    name varchar(255),
+    price decimal(10,2),
+    category varchar(255),
+    description text,
+    specifications json,
+    created_timestamp timestamp,
```

Comment:
Do we not need to mention a format here? What is the format in which this will return the date?